build/testing/sauce-labs/test-browsers.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * @name 
+ * Test Browsers
+ * 
+ * @description 
+ * Cross browser testing with Selenium Webdriver (browser configurations) and Appium (device configurations) for testing on Sauce Labs.
+ * 
+ * @example 
+ * To execute on CLI, run 'node .circleci/run-browser-beta-tests.ts' from root directory.
+ * 
+ * @requires
+ * You must create environment variables if executing this script locally using CLI. This can be done 
+ * by adding to your '~/.bashrc' file.
+ * 
+ * export SAUCE_LABS_USER=[some value]
+ * export SAUCE_LABS_USER=[some value] 
+ *
+ * CircleCI has these values included as environment variables
+ * 
+ * @see
+ * To understand in detail:
+ * Getting Started: https://help.crossbrowsertesting.com/selenium-testing/getting-started/javascript/
+ * Scripting docs: https://www.seleniumhq.org/docs/03_webdriver.jsp#chapter03-reference 
+ */
+
+const { Builder } = require("selenium-webdriver");
+const chalk = require('chalk');
+const { Configure, Phases } = require("../config-browsers.js");
+const { spawn } = require('child_process');
+const git = spawn('git', ['rev-parse', '--abbrev-ref', 'HEAD']);
+
+// Retrive user/key from environment variables
+const username = process.env.SAUCE_LABS_USER;
+const accessKey = process.env.SAUCE_LABS_KEY;
+const domain = "https://msft-docs.fast-dna.net";
+
+// Configure to connect to remote hub
+const remoteHub = "http://" + username + ":" + accessKey + "@ondemand.saucelabs.com:80/wd/hub";
+
+/**
+ * Get configuration based on the accepted Phase argument
+ * @param {Phases} phases
+ * @returns {Configuration based on phases}
+ */
+const getConfiguration = (phases) => {
+
+    // Parse command arguments and return valid configs
+    try {
+        switch (phases) {
+            
+            case "alpha":
+                return new Configure(Phases.Alpha);
+                
+            case "beta":
+                return new Configure(Phases.Beta);
+                
+            case "release":
+                return new Configure(Phases.Release);
+                
+            default:
+                console.log(chalk.red("Invalid Argument : must be 'alpha', 'beta', or 'release'"));
+                return process.exit(1);
+        }
+
+    } catch(err) {
+        console.error(err)
+    }
+}
+
+// Assign browser configurations to an array
+let browsers = getConfiguration(process.argv[2]);
+
+/**
+ * Run tests on Sauce Labs
+ * @param {string} branchName 
+ */
+function Test(branchName) {

```

Comment:
nit - we typically only capitalize classes - maybe rename to `testBranch` to be more descriptive

Reply:
Good catch. I will update.